### ISSUE TITLE
New integrated workbench selector 'Toolbar' shows a white line

### ISSUE THREAD

**Issue by DrD4ffy** [obelisk79/OpenDark#76]
The white line under the buttons looks more like a glitch than a sensible separation:
![image](https://github.com/obelisk79/OpenDark/assets/156334536/1ad72815-05d1-4878-bd34-31c35edc0892)

**Comment by obelisk79** [comment]
Yeah not that's not there on purpose. This is tabbar add-on and not selector, right?

**Comment by DrD4ffy** [comment]
Would it be possible to get something like this:

![Tabs](https://github.com/obelisk79/OpenDark/assets/156334536/149904ac-b780-40f4-a1b7-1c4378a84900)

A bit less button, a bit more tab style.

**Comment by obelisk79** [comment]
Like your mockup? not quite, but I made the style decision to go the button route because tabs are more of a ribbon bar interface element, the button style is more discrete, but at the same time this is really a matter of personal preference.
